datamodel/src/rendering/BufferObject.cpp

```diff
@@ -3,7 +3,60 @@
 #include <boost/log/trivial.hpp>
 
 namespace rcbe::rendering {
-VertexBufferObject::VertexBufferObject(const std::vector<rcbe::geometry::Mesh>& meshes) {
+
+VertexBufferObject::VertexArrayObject::VertexArrayObject() {
+    glGenVertexArrays(1, &id_);
```

Comment:
You should actually handle return codes.

Reply:
They all return void. Error codes are handled in main renderer routine.